doc/BMI_MODELS.md

```diff
@@ -45,7 +57,11 @@ The catchment entry in the formulation/realization config must be set to used th
 
 Valid name values for the currently implemented BMI formulation types are:
 
+* `bmi_c++`
 * `bmi_c`
+* `bmi_fortran`
+* `bmi_python`
```

Comment:
Notably, the docs don't cover this yet... though it's supported... should it be removed?

Reply:
Those (`bmi_fortran`, `bmi_python`, `bmi_multi`) should have been added previously, so adding them now is appropriate.  A section on details of Python module usage is probably also still needed, but can be added separately.